src/zjs_modules.c

```diff
@@ -206,9 +208,53 @@ static ZJS_DECL_FUNC(native_require_handler)
     return jerry_acquire_value(found_obj);
 }
 
+// native eval handler
+static ZJS_DECL_FUNC(native_eval_handler)
+{
+    return zjs_error("eval not supported");
+}
+
+// native print handler
+static ZJS_DECL_FUNC(native_print_handler)
+{
+    if (argc < 1 || !jerry_value_is_string(argv[0]))
+        return zjs_error("print: missing string argument");
+
+    jerry_size_t size = 0;
+    char *str = zjs_alloc_from_jstring(argv[0], &size);
+    if (!str)
+        return zjs_error("print: out of memory");
+
+    ZJS_PRINT("%s\n", str);
+    zjs_free(str);
+    return ZJS_UNDEFINED;
+}
+
+static ZJS_DECL_FUNC(stop_js_handler)
+{
+    #ifdef CONFIG_BOARD_ARDUINO_101
+    zjs_ipm_free_callbacks();
+    #endif
+    zjs_modules_cleanup();
+    jerry_cleanup();
+    return ZJS_UNDEFINED;
+}
+
 void zjs_modules_init()
 {
+    // Add module.exports to global namespace
     ZVAL global_obj = jerry_get_global_object();
```

Comment:
@brianjjones, the reason to use ZVAL is that it's auto-released, you don't have to call jerry_release() on them, that's why I mentioned in your PR